legend-engine-xt-relationalStore-mutation-executionPlan-test/src/main/resources/core_relational_mutation_execution_test/relational/mutation/testRelationalMutation.pure

```diff
@@ -0,0 +1,299 @@
+// Copyright 2020 Goldman Sachs

```

Comment:
Move these tests to legend-engine-server/legend-engine-query-pure?

Reply:
This would introduce a cycle, as these tests depend on the legend compile function, which requires the relational grammar/pure